src/test/groovy/org/mockito/release/internal/gradle/GitPluginTest.groovy

```diff
@@ -0,0 +1,28 @@
+package org.mockito.release.internal.gradle
+
+import org.gradle.testfixtures.ProjectBuilder
+import spock.lang.Specification
+
+
+class GitPluginTest extends Specification {
+
+    def project = new ProjectBuilder().build()
+
+    def setup() {
+        project.plugins.apply(GitPlugin.class)
+    }
+
+    def "default commitMessagePostfix"() {
+        expect:
+        project.releasing.git.commitMessagePostfix == " [ci skip]"
+    }
+
+    def "custom commitMessagePostfix"() {
+        def buildNo = 1234
+        project.releasing.git.commitMessagePostfix = buildNo? " by CI build $buildNo [ci skip-release]" : " [ci skip-release]"
```

Comment:
to keep the tests as simple as possible, lets avoid conditional logic from tests. I suggest to just configure the postfix to fixed value and assert on it.